Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/SlidesLimitSwitchTest.java

```diff
@@ -1,4 +1,4 @@
-package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+/*package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
```

Comment:
Don't just comment out this file. Use the disabled flag.

Reply:
Resolved